README.md

```diff
@@ -31,6 +31,8 @@
 
 - 📄 **JSON Parsing and Generation**: Offers detailed parsing and generation capabilities for nftables rulesets in JSON format, enabling seamless integration and manipulation of rulesets.
 
+- **JSON Schema generation for NFtables**: Allows to create and export a JSON Schema for further usage derived from the explicit Rust types.
```

Comment:
For the sake of consistency, I think there is an emoji missing here ;) How about 💾? Or do you maybe have a better idea?

```suggestion
- 💾 **JSON Schema generation for NFtables**: Allows to create and export a JSON Schema for further usage derived from the explicit Rust types.
```

Reply:
I feel that, it is a touch choice ;)